docs/sources/administration/plugin-management/index.md

```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
 
 ### Install a plugin
 
-To install a plugin:
+To install a plugin you have multiple possibilities, the regular way is using Grafana UI.
 
 1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
 1. Browse and find a plugin.
 1. Click the plugin's logo.
 1. Click **Install**.
 
-When the update is complete, you'll see a confirmation message that the installation was successful.
-
-### Update a plugin
+There are also additional ways to install plugins depending on your setup.
 
-To update a plugin:
-
-1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
-1. Click the **Installed** filter to show only installed plugins.
-1. Click the plugin's logo.
-1. Click **Update**.
+#### Install a plugin using grafana cli
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins using a command line. For more information about Grafana CLI plugin commands, refer to [Plugin commands]({{< relref "../../cli/#plugins-commands" >}}).
 
-When the update is complete, you'll see a confirmation message that the update was successful.
 
-### Uninstall a plugin
+#### Install a plugin from a zip file
+This way is usually used for plugins not distributed via plugins catalog or in environments without internet connection.
 
-To uninstall a plugin:
+Download the archive containing the plugin assets and install it by extracting the archive into  plugin directory. For example:
 
-1. In Grafana, click **Administration > Plugins and data > Plugins** in the side navigation menu to view all plugins.
-1. Click the plugin's logo.
-1. Click the **Installed** filter to show only installed plugins.
-1. Click **Uninstall**.
-
-When the update is complete, you'll see a confirmation message that the uninstall was successful.
-
-## Install Grafana plugins
-
-Grafana supports data source, panel, and app plugins.
+```bash
+unzip my-plugin-0.2.0.zip -d YOUR_PLUGIN_DIR/my-plugin
+```
 
-1. In a web browser, navigate to the [Grafana plugin catalog](https://grafana.com/plugins) and find a plugin that you want to install.
-1. Click the plugin, and then click the **Installation** tab.
+The path to the plugin directory is defined in the configuration file. For more information, refer to [Configuration]({{< relref "../../setup-grafana/configure-grafana/#plugins" >}}).
 
-### Install plugin on Grafana Cloud
+#### Install a plugin in air-gapped environment
+Plugin installation usually requires an internet connection, you can check which endpoints are being used on your instance when installing a plugin regularly and add them to allowlist of your instance.
```

Comment:
Plugin installation usually requires an internet connection. You can check which endpoints are used during the installation on your instance and add them to your instance’s allowlist.

Reply:
thanks, adjusted